src/main/java/com/kodilla/ecommercee/domain/groups/Group.java

```diff
@@ -10,20 +8,20 @@
 
 import javax.persistence.*;
 import javax.validation.constraints.NotNull;
-import java.util.ArrayList;
 import java.util.List;
 
 @AllArgsConstructor
 @NoArgsConstructor
```

Comment:
Nie wiem czy Hibernate zadziała Tobie z @NoArgsConstructor mi coś nie działało dlatego musiałem zadeklarować pusty, ale to może dlatego, że ja adnotacje piszę nad getterami i setterami. Jeżeli test przeszedł to chyba Tobie diząła

Reply:
sam jestem ciekawy, czy takie rozwiązanie będzie później działać poprawne :)